packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5133,6 +5133,981 @@ curl -v -X GET \
+  "e":"AQAB",
+  "n":"AJncrKuine49_CEVR4GPn.....zOrouIUCSMlRL0HU=",
+  "status": "INACTIVE",
+  "created": "2022-01-01 21:51:14.0",
+  "_links" : {
+    "activate": {
+      "href": "https://${yourOktaDomain}/api/v1/apps/0oad5lTSBOMUBOBVVQSC/credentials/jwks/pks055raeabcdef5rpg6/lifecycle/activate",
+      "hints": {
+        "allow": [
+          "POST"
+        ]
+      }
+    },
+    "delete": {
+      "href": "https://${yourOktaDomain}/api/v1/apps/0oad5lTSBOMUBOBVVQSC/credentials/jwks/pks055raeabcdef5rpg6/delete",
+      "hints": {
+        "allow": [
+          "DELETE"
+        ]
+      }
+    }
+  }
+}
+```
+If a secret isn't found for the `secretId`, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000007",
+  "errorSummary": "Not found: Resource not found: 2345 (JsonWebKey)",
+  "errorLink": "E0000007",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw"
+}
+```
+
+If the application has only one active JSON Web Key, the `token_endpoint_auth_method` is `private_key_jwt`, and a deactivate request is received for the JSON Web Key, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: JsonWebKey",
+  "errorLink": "E0000001",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw",
+  "errorCauses": [
+    {
+      "errorSummary": "Can't deactivate the only active JSON Web Key when the value for ''token_endpoint_auth_method'' is ''private_key_jwt''."
+    }
+  ]
+}
+```
+
+If the application has only one active JSON Web Key that matches `request_object_signature_algorithm` and a deactivate request is received for the JSON Web Key, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: JsonWebKey",
+  "errorLink": "E0000001",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw",
+  "errorCauses": [
+    {
+      "errorSummary": "The ''jwks'' must contain at least one key with an algorithm matching the ''request_object_signature_algorithm''."
+    }
+  ]
+}
+```
+
+### Delete a JSON Web Key
+
+<ApiOperation method="delete" url="/api/v1/apps/${applicationId}/credentials/jwks/{id}"/>
+
+Deletes a specific JSON Web Key by `id`
+
+##### Request parameters
+
+| Parameter     | Description                                                                     | Param Type | DataType                                      | Required | Default |
+| ------------- | ------------------------------------------------------------------------------- | ---------- | --------------------------------------------- | -------- | ------- |
+| applicationId | Unique key of an [Application](#application-object)                             | URL        | String                                        | TRUE     |         |
+| id            | Unique key of a [JSON Web Key](#json-web-key-response-object)                   | URL        | String                                        | TRUE     |         |
+
+##### Response parameters
+
+An empty JSON object `{}`
```

Comment:
As mentioned above, no response body with 204.

Reply:
Followed https://developer.okta.com/docs/reference/api/apps/#request-parameters-7

Reply:
Same comment in this case, please fix both places if you'd like for them to stay consistent.

Reply:
updated